docs/data/quick-start.md

```diff
@@ -0,0 +1,791 @@
+
+    ```
+
+=== "Node"
+
+    ```ts title="TypeScript"
+
+    import { track } from '@amplitude/analytics-node';
+
+    const eventProperties = {
+      buttonColor: 'primary',
+    };
+
+    track('Button Clicked', eventProperties, {
+      user_id: 'user@amplitude.com',
+    });
+    ```
+
+=== "Android"
+
+    ```kotlin title="Kotlin"
+
+    amplitude.track("eventType", mutableMapOf<String, Any?>("test" to "event property value"))
+    ```
+
+    ```java title="Java"
+    amplitude.trak("eventType", new HashMap() {{
+        put("test", "test event property value");
+    }});
+    ```
+
+=== "iOS"
+
+    ```obj-c title="Objective-C"
+    [[Amplitude instance] logEvent:@"Button Clicked"];
+    ```
+
+    ```swift title="Swift"
+    Amplitude.instance().logEvent("Button Click")
+    ```
+
+=== "Java"
+
+    ```java
+
+    amplitude.logEvent(new Event("Button Clicked", "test_user_id"));
+    ```
+
+=== "Python"
+
+    ```python title="Python"
+    from amplitude import BaseEvent
+
+    amplitude.track(
+        BaseEvent(
+            event_type="type of event",
+            user_id="USER_ID",
+            device_id="DEVICE_ID",
+            event_properties={
+                "source": "notification"
+            }
+    ))
+    ```
+
+=== "React Native"
+
+    ```ts title="TypeScript"
+
+    import { track } from '@amplitude/analytics-react-native';
+
+    const eventProperties = {
+        buttonColor: 'primary',
+    };
+    track('Button Clicked', eventProperties);
+    ```
+
+=== "Flutter"
+
+    ```dart title="Dart"
+
+    amplitude.logEvent('BUTTON_CLICKED', {"Hover Time": "100ms"});
+    ```
+
+=== "Unity"
+
+    ```c# title="C#"
+    import 'package:amplitude_flutter/amplitude.dart';
+
+    amplitude.logEvent('MyApp startup', eventProperties: {
+      'friend_num': 10,
+      'is_heavy_user': true
+    });
+    ```
+
+=== "Unreal"
+
+    ```c++ title="C++"
+
+    TArray<FAnalyticsEventAttribute> AppendedAttributes;
+    AppendedAttributes.Emplace(TEXT("Test Event Prop key1"), TEXT("Test Event value1"));
+    AppendedAttributes.Emplace(TEXT("Test Event Prop key2"), TEXT("Test Event value2"));
+    FAnalytics::Get().GetDefaultConfiguredProvider()->RecordEvent(TEXT("Game Started"), AppendedAttributes);
+    ```
+
+=== "GO"
+
+    ```go title="GO"
+
+    amplitude.Track(amplitude.Event{
+           EventType: "Button Clicked",
+           EventOptions: amplitude.EventOptions{
+           UserID:   "user-id",
+           DeviceID: "device-id",
+        },
+        EventProperties: map[string]interface{}{"source": "notification"},
+    })
+
+    ```
+
+## 3. Check for success
+#### Ingestion Debugger
+    1. Login to you Amplitude Analytics Orgnization.
+    2. Click the `Data Source tab` at the bottom of the navigation bar
+    3. Choose the `Ingestion Debugger` section. You can check the Requests or which User or Device has been throttled.
+
+Learn more about [Ingestion Debugger](https://help.amplitude.com/hc/en-us/articles/360044835531).
```

Comment:
Is there a way to avoid these extra lines for learn more for each option?

@caseyamp is there some type of annotation/subscript we can attach to the titles instead?

Reply:
Talked to @caseyamp , we cannot have annotation on the title. And she thought random links are jarring for some readers, so where possible we need to provide more info about what they're actually clicking on.

Reply:
<img width="780" alt="Screen Shot 2022-09-02 at 3 07 07 PM" src="https://user-images.githubusercontent.com/13028329/188241239-014407a9-1478-4dad-8e40-250c8d7a80c1.png">
I add a <br> instead of having a new line. 
But I still prefer the one with extra space. What do you think?

Reply:
I like the extra space as well.